modules/appeals_api/spec/requests/metadata_request_spec.rb

```diff
@@ -100,54 +100,54 @@
           expect(upstream_service['details']['time']).to eq('2020-09-21T00:00:00Z')
         end
       end
+    end
+
+    context 'v1' do
```

Comment:
this just shifts v1 context out next to v0, instead of inside it for some strange reason